src/main/java/com/epam/training/array/ArrayLogic.java

```diff
@@ -1,16 +1,85 @@
 package com.epam.training.array;
 
+import java.util.Arrays;
```

Comment:
@judita-b Could you remove unused imports please

Reply:
I have removed unused imports